### PR TITLE
fix docs

### PR DESCRIPTION
**Pull request by Trinkle23897**
- fix css style error
- fix mujoco benchmark result

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ Welcome to Tianshou!
 
 Here is Tianshou's other features:
 
-* Elegant framework, using only ~2000 lines of code
+* Elegant framework, using only ~3000 lines of code
```
Comment (araffin) [inline docs/index.rst:36]: out of curiosity, how did you compute that number? 
using `tokei tianshou` (https://github.com/XAMPPRocky/tokei), I get:
```
===============================================================================
 Language            Files        Lines         Code     Comments       Blanks
===============================================================================
 Python                 62         7964         7066          236          662
===============================================================================
 Total                  62         7964         7066          236          662
===============================================================================
```

and on the complete repo:

```
===============================================================================
 Language            Files        Lines         Code     Comments       Blanks
===============================================================================
 Autoconf                1            1            1            0            0
 CSS                     1          139           98           11           30
 JavaScript              2          131          117           10            4
 JSON                   10           18           18            0            0
 Makefile                1           20            9            7            4
 Python                127        17554        14995          878         1681
 ReStructuredText       15         2805         2104            0          701
 Shell                   1           12            9            1            2
 Plain Text              1            7            0            7            0
-------------------------------------------------------------------------------
 Markdown                7          721            0          511          210
 |- BASH                 2           14           11            3            0
 |- Python               1           45           42            3            0
 (Total)                            780           53          517          210
===============================================================================
 Total                 166        21408        17351         1425         2632
===============================================================================

```

Comment (Trinkle23897) [reply docs/index.rst:36]: I use https://app.codecov.io/gh/thu-ml/tianshou
image

Comment (Trinkle23897) [reply docs/index.rst:36]: Btw, the table in readme will be updated soon... sorry about that

Comment (araffin) [reply docs/index.rst:36]: ok, thanks =) (the number of files is not even the same ^^")

Comment (araffin) [reply docs/index.rst:36]: oh, I see, it counts only the files that are declared to be covered by tests, no?

Comment (Trinkle23897) [reply docs/index.rst:36]: Yep

**Pull request merged**